news/2020-02-03-Micro_Profile_OpenTracing_Comes_To_WildFly.adoc

```diff
@@ -0,0 +1,113 @@
+= Eclipse MicroProfile OpenTracing comes to WildFly =
+ehsavoie
+2020-02-17
+:revdate: 2020-02-03 08:00:00 -0600
```

Comment:
Update this date to match; I didn't do that for https://wildfly.org/news/2020/01/17/Eclipse-MicroProfile-3.3-in-WildFly-19/ and accidentally generated a Jan 17 url for a blog posted in March. And then I sent out mails and tweets and etc so I couldn't fix it!

Reply:
I updated all the dates